cdap-gateway/src/main/java/co/cask/cdap/gateway/router/NettyRouter.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright © 2014-2015 Cask Data, Inc.
+ * Copyright © 2016 Cask Data, Inc.
```

Comment:
I did this manually, and IIRC it should be just "Copyright © {YEAR}" instead of a range.